src/basic/memory-util.h

```diff
@@ -88,9 +89,7 @@ static inline void* erase_and_free(void *p) {
 
         l = malloc_usable_size(p);
         explicit_bzero_safe(p, l);
-        free(p);
-
-        return NULL;
+        return mfree(p);
```

Comment:
I'm not sure about this one, if it's worth it :thinking:

Reply:
looks good